pages/api/discount-usage.ts

```diff
@@ -0,0 +1,138 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import "@shopify/shopify-api/adapters/node";
+import { type Address } from "viem";
+import { getAddressFromQuery } from "@/utils/getAddressFromQuery";
+import { env } from "@/config/env.mjs";
+import { LATEST_API_VERSION, shopifyApi } from "@shopify/shopify-api";
+
+interface DiscountReturnType {
+  codeDiscountNodes: {
+    edges: Array<{
+      node: {
+        id: string;
+        codeDiscount: {
+          title: string;
+          status: string;
+          usageLimit: number;
+          asyncUsageCount: number;
+        };
+      };
+    }>;
+  };
+}
+
+const SHOPIFY_API_ACCESS_TOKEN = env.SHOPIFY_API_ACCESS_TOKEN;
```

Comment:
do we have a file that contains all consts? If yes can we move this to that place?

Reply:
not yet so I'm moving all shopify-related consts to a separate file. I will make a new PR with the rest of the consts!